lesson-02-assignment

```diff
@@ -0,0 +1,93 @@
+#Question 1
+    #create courses 1
+    new_course = CodingClass.all
+    new_course.each do |x|
+    Course.create(coding_class_id:x.id, trimester_id:8, max_enrollment:25)
+    end
+
+    #create courses 2
+    CodingClass.all.each do |x|
```

Comment:
just a note! It's probably best practice to start naming your blocks something other than `x` to make it easier on you in the future. In this case you can name it `coding_class`.